src/run/run.c

```diff
@@ -637,23 +647,58 @@ static int transient_unit_set_properties(sd_bus_message *m, UnitType t, char **p
 }
 
 static int transient_cgroup_set_properties(sd_bus_message *m) {
+        _cleanup_free_ char *slice = NULL;
+        _cleanup_free_ char *name = NULL;
         int r;
         assert(m);
 
-        if (!isempty(arg_slice)) {
-                _cleanup_free_ char *slice = NULL;
+        if ((int) !!arg_slice + (int) !!arg_slice_inherit_set > 1)
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                       "--slice may not be combined with --slice-inherit.");
```

Comment:
we generally try to suffix options that take options with `=` in docs and log msgs, hence `--slice` → `--slice=` here please

Reply:
oh I didn't notice, again I looked at other errors above, but they didn't take argument I guess.